sqlmesh/dbt/manifest.py

```diff
@@ -172,13 +172,15 @@ def _load_tests(self) -> None:
             )
 
             test_model = _test_model(node)
+            node_config = _node_base_config(node)
+            node_config["database"] = node.database or self.target.database
```

Comment:
why do we do this?

Reply:
For singular tests, dbt returns the test database in the "this" obj. This is to match it.

Reply:
So the problem with this is that we no longer know whether the `database` was actually overridden for any given model.

Reply:
This is only for a `test` node.